app/models/builders/manual_builder.rb

```diff
@@ -24,10 +18,14 @@ def call(attrs)
       use_originally_published_at_for_public_timestamp: true,
     }
 
-    factory.call(default_attrs.merge(attrs))
+    attrs = default_attrs.merge(attrs)
```

Comment:
I've changed the code to `manual_attrs = default_attrs.merge(attrs)` to avoid the confusion.

I haven't inlined `default_attrs.merge(attrs)` in `Manual.new` because that would mean we'd be reading the `sections` and `removed_sections` from `attrs` but instantiating the object with `default_attrs.merge(attrs)`. While this wouldn't actually change the behaviour it would mean the code was slightly different than the code I took from the factory that I'm inlining.